app/services/katello/pulp/pulp_content_unit.rb

```diff
@@ -86,6 +86,17 @@ def backend_data
       def fetch_backend_data
         self.class.pulp_data(self.uuid)
       end
+
+      def self.remove(repo_id, uuids = nil)
```

Comment:
can you change this method to take in the repository object instead of repo_id?  In pulp 3 land, it may not be '.repo_id' that is needed.